examples/.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We have avoided to commit this file to the repos so far so each dev can customize it to their preferences.

Let me add it to .gitignore.